README.md

```diff
@@ -4,3 +4,62 @@
 
 This repository provides support for applications using the [Google Java Client Libraries](https://github.com/googleapis/google-cloud-java)
 to be built as [GraalVM Native Images](https://www.graalvm.org/docs/reference-manual/native-image).
+
+## Quick Setup
+
+First, add the Sonatype Snapshots repository to your project build file.
+
+```
+<repositories>
+    <repository>
+        <id>snapshots-repo</id>
+        <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+        <releases><enabled>false</enabled></releases>
+        <snapshots><enabled>true</enabled></snapshots>
+    </repository>
+</repositories>
+```
+
+After adding the snapshots repository, you can add the `google-cloud-graalvm-support` dependency
```

Comment:
Should we follow Spring's 1-line-per-sentence convention?

Reply:
Ok, that sounds good. Done.